packages/webcrypto-ed25519-polyfill/src/index.ts

```diff
@@ -116,4 +116,30 @@ if (!__BROWSER__ || globalThis.isSecureContext) {
             throw new TypeError('No native `verify` function exists to handle this call');
         }
     }) as SubtleCrypto['verify'];
+
+    /**
+     * Override `SubtleCrypto#importKey`
+     */
+    const originalImportKey = originalSubtleCrypto.importKey as
+      | SubtleCrypto["importKey"]
+      | undefined;
+    originalSubtleCrypto.importKey = (async (
+      ...args: Parameters<SubtleCrypto["importKey"]>
+    ) => {
+      const [format, keyData, algorithm, extractable, keyUsages] = args;
+      if (algorithm === "Ed25519") {
+        return await importKeyPolyfill(
+          format,
+          keyData,
+          extractable,
+          keyUsages as any
```

Comment:
you might be able to sneak around this `any` check by following what some of the other methods are doing, ie:

https://github.com/solana-labs/solana-web3.js/blob/eb5fd160f78c17f360d679861f487b42c693bd69/packages/webcrypto-ed25519-polyfill/src/index.ts#L95